### DISCUSSION TITLE
Is there a way to add custom ffmpeg encoder flags?

### DISCUSSION THREAD

**Discussion by RisaKirisu** [LizardByte/community#277] [Sunshine] [answered]
I was wondering if I can experiment with different custom ffmpeg options such as --no-sao to decrease the some smoothing issue in HEVC, specifically on NVENC. I don't see a place to do this in the settings. Is there I way to do it in the code?

**Comment by ns6089** [comment, marked as answer]
NvENC doesn't expose SAO option in their API, ffmpeg or not.

**Reply by RisaKirisu** [reply]
Wasn't aware of the fact. That's silly of me. Thanks!
